docs/digger/installation/digger-install-intro.asciidoc

```diff
@@ -1,12 +1,22 @@
-== Digger Installation Introduction
+== AeroGear Digger Installation
+//Please DO NOT use 'Overview' as a section heading anywhere http://stylepedia.net/#sect-Red_Hat_Technical_Publications-Writing_Style_Guide-Overall_Book_Design-Unused_Heading_Titles
+https://github.com/aerogear/aerogear-digger-installer[Digger Installer^] - GitHub repo with Ansible playbook used for Digger installation.
 
-The AeroGear Digger project is a tool for building mobile applications using OpenShift containers and Jenkins CI/CD.
+The AeroGear Digger project allows you to setup a Jenkins CI/CD server for building mobile applications using OpenShift containers.
 
-It is made of following components:
+For more details about Ansible playbooks and variables, please refer to http://docs.ansible.com/ansible/playbooks_variables.html[Ansible Playbook Variables Docs^].
 
-* A Jenkins server that is used to run the build tasks
-* A Docker-formated image with all the required tools to build Android applications
-* macOS servers with pre-installed tools to build iOS applications (optional)
-* An http://docs.ansible.com/ansible/index.html[Ansible^] based automation tool, https://github.com/aerogear/aerogear-digger-installer[aerogear-digger-installer^] used to install and configure all the components
-* A nagios server for monitoring the digger components
-* A suite of sample Jenkinsfiles to demonstrate how to build mobile applications using Jenkins pipeline
+It contains the following components:
+
+* A Jenkins server, used to run the build tasks
+* An http://docs.ansible.com/ansible/index.html[Ansible^] based automation tool used to install and configure components for Android and iOS application building
```

Comment:
Why are we removing the link to digger installer here?

Reply:
moved to line 3, not removed

Reply:
@pmacko1 what is the motivation for removing it from the list of components?

Reply:
@laurafitzgerald now i understand :). I thought that using the link in "Digger Installer - GitHub repo with Ansible playbook used for Digger installation." and here would be redundant. 

Also i should rename Digger Installer - GitHub repo to aerogear-digger-installer - GitHub repo

This needs reviewing, to tell the readers that this is an Ansible based installation tool that installs the following things: ... ... Will commit changes, from those it will be more clear.

Reply:
idea: keep only installation related stuff here. Put overview what digger actually contains in splash or some "overview" short doc

Reply:
@laurafitzgerald why is installer removed from the list of components?

If we were talking about Digger as a complete project, then the installer would part part of the list.
This is an installation guide that tells user what will be installed by the installer, so I would prefer to leave the "installer" out of this list and only list the components that are installed with the installer.

From previous comment ^ an overview of the repos/product, maybe fancy diagrams (i have templates, maybe can do that), would help. wdyt?

Reply:
@pmacko1 ok so I see we renamed the heading from introduction -> installation. Do we now have some introduction section somewhere?

Reply:
@laurafitzgerald no, i'm not aware of it, that is. i don't know about the plan how the splash and final doc set will look like. I see that some portion of info was put in the installation guide that maybe should be included in some "overview" doc mentioned in the previous comment.

Reply:
@laurafitzgerald https://issues.jboss.org/browse/AGDIGGER-268

jira'ed